examples/scraping_beam/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _examples-scraping:
+
+Expanding Beam Scraping Against a Vacuum Pipe
+==============================================
+
+This example describes a coasting bunch, expanding transversely and encountering the aperture defined by the vacuum pipe.
+Space charge is neglected, making the problem analytically soluble.
+
+We use a cold (zero emittance) 250 MeV proton bunch whose
+initial distribution is a uniformly-populated cylinder of transverse radius rb = 2 mm with zero momentum spread.
+
+The beam propagates in a drift with a vacuum chamber radius of R = 3.5 mm.
+
+To generate an expanding beam, a linear map is first applied.  This map applies a radial kick to each particle that is proportional to the particle's initial distance from the axis.
+This induces a phase space correlation within the beam, such that px = k*x and py = k*y, similar to what would be induced by a space charge kick.
+
+The beam remains cylindrical with zero emittance during its evolution in a 6 m drift.
+In the absence of an aperture, the beam radius evolves as :math:`rb(s) = rb(1 + ks)`.
+In the presence of an aperture, particles are lost during the transverse expansion.  The fraction of charge remaining after a distance s is given by:
+
+:math:`Q_s/Q_0 = \min{1,R^2/(rb^2(1+sk)^2)}`.
```

Comment:
If you like, you can as in LaTeX also create math blocks:
```suggestion
.. math::

   Q_s/Q_0 = \min{1,R^2/(rb^2(1+sk)^2)}.
```
https://sphinx-rtd-trial.readthedocs.io/en/latest/ext/math.html